src/scss/04-patterns/03-interaction/_action-sheet.scss

```diff
@@ -149,3 +149,24 @@
 		}
 	}
 }
+
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	.action-sheet-container--visible .action-sheet {
+		background-color: rgba(0, 0, 0, 1);
```

Comment:
If this is used in more places, maybe we can add a new css variable with this transparent background

Reply:
Changed to the `--neutral-0`, OS will put it as a dark color.